designs/2021-suppression-support/README.md

```diff
@@ -422,7 +422,7 @@ NOTE that the kind `directive` should be converted to `inSource` for SARIF accor
 
 ### Return `SuppressedLintMessage` in `Linter#verify` and `Linter#verifyAndFix`
 
-Since we have a new property `suppressedMessages` in `LintResult`, the output of `Linter#verify` and `Linter#verifyAndFix` will also have `suppressedMessages`.
+Since we have a new property `suppressedMessages` in `LintResult`, the output of `Linter#verify` and `Linter#verifyAndFix` will also have `suppressedMessages`. That is, `Linter#verify` will return `{messages: LintMessage[], suppressedMessages: SuppressedLintMessage[]}`, and `Linter#verifyAndFix` will return `{fixed: boolean, messages: LintMessage[], output: string, suppressedMessages: SuppressedLintMessage[]}`.
```

Comment:
We can’t change these methods to return an object. That would be a significant breaking change to the API. The options are:

1. Return an array with an added suppressedMessages property.
2. Add a getSupressedMessages() method to Linter that can retrieve the suppressed messages from the previous run.

Reply:
Arrays with added properties are very weird; aside from regexp match objects, they rarely happen, and that's a good thing - out of those two options, the second seems far better (and also avoids penalizing the common case of not needing this info)